scipy/sparse/linalg/eigen/arpack/arpack.py

```diff
@@ -1361,7 +1361,8 @@ def eigsh(A, k=6, M=None, sigma=None, which='LM', v0=None,
     Returns
     -------
     w : array
-        Array of k eigenvalues
+        Array of k eigenvalues.
+
```

Comment:
We don't need empty lines between the items in the same section.

Reply:
Yes, It seems that there is a blank line in editing process. I will correct your point.